### PR TITLE
docs: fix: clear up some issues in docs about modules

### PR DESCRIPTION
**Pull request by xynydev**
- better distinction between modules and module configuration
- example for module configuration external from recipe.yml

### CODE REVIEW

File: config/README.md [hunk 1]
```diff
@@ -16,16 +16,27 @@ At the top of the recipe, there are four *mandatory* configuration options.
...
 
-This repository fetches some useful default modules from [`ublue-os/bling`](https://github.com/ublue-os/bling/), like [`rpm-ostree`](../modules/rpm-ostree) for pseudo-declarative package management, [`bling`](../modules/bling) for pulling extra components from [`ublue-os/bling`](https://github.com/ublue-os/bling), and [`files`](../modules/files) for copying files from the `config/files/` directory into your image. For a comprehensive list of modules and their in-depth documentation, check out [the modules page on the website](https://universal-blue.org/tinker/modules/).
+This repository fetches some useful default modules from [`ublue-os/bling`](https://github.com/ublue-os/bling/), like [`rpm-ostree`](../modules/rpm-ostree) for pseudo-declarative package management, [`bling`](../modules/bling) for pulling extra components from [`ublue-os/bling`](https://github.com/ublue-os/bling), and [`files`](../modules/files) for copying files from the `config/files/` directory into your image.
```
Comment (gerblesh) [inline config/README.md]: Seems like there's some dead links to the module documentation here, when clicking it just leads to a 404

File: config/README.md [hunk 2]
```diff
@@ -16,16 +16,27 @@ At the top of the recipe, there are four *mandatory* configuration options.
...
 To build multiple images, you need to create another recipe.yml file, which you should name based on what kind of image you want it to build. Then, edit the [`build.yml`](../.github/workflows/build.yml) file. Inside the file, under `jobs: strategy: matrix:`, there's a list of recipe files to build images, which you need to add your new recipe file to. These should be paths to files inside the `config` directory.
 
-Module configuration can be included from other files using the `from-file` syntax. The valye should be a path to a file inside the `config` directory. For example, the following snippet could be used to include the configuration for installing a set of packages common to multiple images.
+Module configuration can be included from other files using the `from-file` syntax. The value should be a path to a file inside the `config` directory. For example, the following snippet could be used to include the configuration for installing a set of packages common to multiple images.
 ```yml
 modules:
   - from-file: common-packages.yml
+```
+And inside common-packages.yml
+```yml
```
Comment (gerblesh) [inline config/README.md]: i'd use `yaml` instead of `yml` for syntax highlighting

File: config/README.md (hunk 1 above)
Comment (xynydev) [reply config/README.md]: Ahh, good catch

File: config/README.md (hunk 2 above)
Comment (xynydev) [reply config/README.md]: changed it, but curious abt why?

Comment (gerblesh) [reply config/README.md]: when I checked the docs on the website, yml doesn't have syntax highlighting, yaml does. Idk, probably just a weird thing with mkdocs

File: README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Don't worry, it only requires some basic knowledge about using the terminal and
 After setup, it is recommended you update this README to describe your custom image.
 
 > **Note**
-> Everywhere in this repository, make sure to replace `ublue-os/startingpoint` with the details of your own repository. Unless you used [`create-ublue-image`](https://github.com/EinoHR/create-ublue-image), in which case the previous repo identifier should already be your repo's details.
+> Everywhere in this repository, make sure to replace `ublue-os/startingpoint` with the details of your own repository. Unless you used on of the automatic repository setup tools in which case the previous repo identifier should already be your repo's details.
```
Comment (RoyalOughtness) [inline README.md]: typo: one of

**Review by gerblesh — APPROVED** [review]

File: config/README.md [hunk 4]
```diff
@@ -16,16 +16,27 @@ At the top of the recipe, there are four *mandatory* configuration options.
...
 To build multiple images, you need to create another recipe.yml file, which you should name based on what kind of image you want it to build. Then, edit the [`build.yml`](../.github/workflows/build.yml) file. Inside the file, under `jobs: strategy: matrix:`, there's a list of recipe files to build images, which you need to add your new recipe file to. These should be paths to files inside the `config` directory.
 
-Module configuration can be included from other files using the `from-file` syntax. The valye should be a path to a file inside the `config` directory. For example, the following snippet could be used to include the configuration for installing a set of packages common to multiple images.
-```yml
+Module configuration can be included from other files using the `from-file` syntax. The value should be a path to a file inside the `config` directory. For example, the following snippet could be used to include the configuration for installing a set of packages common to multiple images.
+```yaml
 modules:
   - from-file: common-packages.yml
+```
+And inside common-packages.yml
```
Comment (RoyalOughtness) [inline config/README.md]: maybe specify that common-packages.yml is /config/common-packages.yml

Comment (xynydev) [reply config/README.md]: Sure, I'll update this next time I go to the desktop.

**Pull request merged**